### PR TITLE
Add uplink decoder for Comtac AG LPN KM

### PR DESCRIPTION
**Pull request by xExecutioner**
#### Summary

added uplink decoder and yaml tester for comtac AG LPN KM

#### Changes

- added uplink decoder and yaml tester for comtac AG LPN KM

#### Notes for Reviewers

I was not able to validate the files due to #145. Therefore, I start this pull request to use the internal validation

#### Release Notes

- ...

### CODE REVIEW

**Comment by cBashTN** [comment]
Hoi Manuel

In lpn-km-codec.yaml you specify the decoder file with  ```fileName: lpn-km.js``` but the file is named **lpn_km.js**. You might want to check this.

**Comment by xExecutioner** [comment]
> Hoi Manuel
> 
> In lpn-km-codec.yaml you specify the decoder file with `fileName: lpn-km.js` but the file is named **lpn_km.js**. You might want to check this.

Thanks Sebastian,

As I am a first time contributor, it seems I have to wait for an approveral first. I can't see any validation results yet.

File: vendor/comtac/lpn-km-codec.yaml [hunk 1]
```diff
@@ -0,0 +1,112 @@
+# Uplink decoder decodes binary data uplink into a JSON object (optional)
+# For documentation on writing encoders and decoders, see: https://thethingsstack.io/integrations/payload-formatters/javascript/
+uplinkDecoder:
+  fileName: lpn-km.js
```
Comment (Jaime-Trinidad) [inline vendor/comtac/lpn-km-codec.yaml:4]: hi @xExecutioner please check your.js file name, the name of the file is lpn_km.js`

**Review by Jaime-Trinidad — COMMENTED** [review]
files names now correspond

**Comment by johanstokking** [comment]
@xExecutioner I ran the CI just now. This fails:

```
comtac: lpn-km: EU863-870: payload codec lpn-km-codec invalid
./vendor/comtac/lpn-km.js:decodeUplink: output {"data":{"decoder":{"info":"TTN/TTI","version":"000"},"payload":{"data":{"digitalInputs":[{"cot":{"cyclic":false,"event":false,"interrogation":true},"info":{"id":1,"type":"singlePointInfo"},"status":{"blocked":false,"state":0},"timestamp":{"string":"01/01/2021, 00:00:01","unix":1609459201}},{"cot":{"cyclic":false,"event":false,"interrogation":true},"info":{"id":2,"type":"singlePointInfo"},"status":{"blocked":false,"state":0},"timestamp":{"string":"01/01/2021, 00:00:01","unix":1609459201}}]},"device":{"batteryLevel":"77%","info":{"deviceDesignation":"UNKNOWN","deviceId":16,"deviceManufacturer":"comtac AG","deviceVersion":0}}},"payloadLength":16,"port":20,"portFunction":"DI_DATA"},"warnings":[]} does not match {"data":{"payloadLength":16,"port":20,"portFunction":"DI_DATA","decoder":{"info":"TTN/TTI","version":"000"},"payload":{"data":{"digitalInputs":[{"cot":{"cyclic":false,"event":false,"interrogation":true},"info":{"id":1,"type":"singlePointInfo"},"status":{"blocked":false,"state":0},"timestamp":{"string":"01/01/2021, 00:00:01","unix":1609459201}},{"cot":{"cyclic":false,"event":false,"interrogation":true},"info":{"id":2,"type":"singlePointInfo"},"status":{"blocked":false,"state":0},"timestamp":{"string":"01/01/2021, 00:00:01","unix":1609459201}}]},"device":{"batteryLevel":"77%","info":{"deviceDesignation":"UNKNOWN","deviceId":16,"deviceManufacturer":"comtac AG","deviceVersion":0}}}}}
```

You can also run the validation locally with `make validate` (run `make deps` first)

**Comment by xExecutioner** [comment]
Thanks @johanstokking. I tried, but I was not able due to #145.  I will first try to solve this problem

**Comment by Jaime-Trinidad** [comment]
Hi @xExecutioner could you run make validate?

**Comment by xExecutioner** [comment]
> Hi @xExecutioner could you run make validate?

Yes, I was able to run validation. Newest commit is added now.

**Review by Jaime-Trinidad — APPROVED** [review]
files look fine

**Comment by johanstokking** [comment]
Thanks!

**Pull request merged**